### PR TITLE
Feature/for 2

### PR DESCRIPTION
**Pull request by CrazyPandabier**

### CODE REVIEW

File: formicarium/formicarium/Lidar.py [hunk 1]
```diff
@@ -16,8 +19,9 @@ def __init__(self, collider:ICollider, range: float, xPos: float, yPos: float) -
         self.position = (xPos, yPos)
         self.white = Color(255, 255, 255)
         self.red = Color(255, 0, 0)
+        self.m2p = 3779.5275590551
```
Comment (CasLeeflang) [inline formicarium/formicarium/Lidar.py:22]: we should move variables like this to a seperate config file

File: formicarium/formicarium/Lidar.py [hunk 2]
```diff
@@ -41,10 +45,25 @@ def scan(self, map: Surface) -> LaserScan:
...
+        msg.angle_max = float(2*math.pi)
+        msg.angle_increment = float(2*math.pi / 60)
+        msg.time_increment = float(0)
+        msg.scan_time = float(0)
+        msg.range_min = float(0)
+        msg.range_max = float(self.range)
+        msg.ranges = []
+        msg.intensities = []
+        for point in data:
+            msg.ranges.append(math.sqrt((point[0] - x_0)**2 + (point[1] - y_0)**2)/self.m2p)
```
Comment (CasLeeflang) [inline formicarium/formicarium/Lidar.py]: Does the message include the `inf` values for the laser scans in between the scans with colision?

Comment (CasLeeflang) [reply formicarium/formicarium/Lidar.py]: Maybe we should move this calculation to the for loop above and we don't have to save the points in the `data` list

File: formicarium/formicarium/Lidar.py (hunk 1 above)
Comment (CrazyPandabier) [reply formicarium/formicarium/Lidar.py:22]: yeah should make a seperate ticket for this

**Review by CasLeeflang — APPROVED** [review]

**Pull request merged**